### PR TITLE
homework.py is done, test.py with print_done

### PR DESCRIPTION
**Pull request by haranzha-dmitro**
Здається останній етап домашки

### CODE REVIEW

File: git/homework.py [hunk 1]
```diff
@@ -15,23 +15,23 @@ def is_two_object_has_same_value(first: Any, second: Any) -> bool:
...
+    return type(first) == type(second)
 
 
 def is_two_objects_is_the_same_objects(first: Any, second: Any) -> bool:
     """
     If @first and @second has same type should return True
     In another case should return False
     """
-    pass
+    return id(first) == id(second)
```
Comment (dimigor) [inline git/homework.py]: Take a look at `is` operator

File: git/homework.py [hunk 2]
```diff
@@ -97,14 +103,18 @@ def is_word_in_text(word: str, text: str) -> bool:
         >>> False
 
     """
-    pass
+    return word in text.split()
```
Comment (dimigor) [inline git/homework.py]: you do not need to use `.split()` here

File: git/homework.py [hunk 3]
```diff
@@ -97,14 +103,18 @@ def is_word_in_text(word: str, text: str) -> bool:
...
 
 
 def some_loop_exercise() -> list:
     """
     Use loop to create list that contain int values from 0 to 12 except 6 and 7
     """
-    pass
+    result = []
+    for i in range(0, 13):
+        if (i != 6) & (i != 7):
```
Comment (dimigor) [inline git/homework.py]: use `and` instead of `&`

File: git/homework.py [hunk 4]
```diff
@@ -127,7 +140,10 @@ def alphabet() -> dict:
         alphabet()
         >>> {"a": 1, "b": 2 ...}
     """
-    pass
+    alf = {}
+    for i in range(ord('a'), ord('z') + 1):
```
Comment (dimigor) [inline git/homework.py]: take a look at `string` library

**Review by dimigor — CHANGES_REQUESTED** [review]

File: git/homework.py (hunk 1 above)
Comment (haranzha-dmitro) [reply git/homework.py]: i know about is operator, but in this case i thought that "id" is better chois... (first is second)

File: git/homework.py (hunk 3 above)
Comment (haranzha-dmitro) [reply git/homework.py]: Ok.

File: git/homework.py (hunk 2 above)
Comment (haranzha-dmitro) [reply git/homework.py]: Ok

**Pull request merged**